src/core/init.js

```diff
@@ -11,37 +11,22 @@ geo.fileReaders = {};
  * Convenient function to define JS inheritance
  */
 //////////////////////////////////////////////////////////////////////////////
-inherit = function (C, P) { // jshint ignore: line
+geo.inherit = function (C, P) { // jshint ignore: line
   "use strict";
 
-  var F = function () {
-  };
+  var F = inherit.func();
   F.prototype = P.prototype;
   C.prototype = new F();
-  C.uber = P.prototype;
   C.prototype.constructor = C;
 };
-
-//////////////////////////////////////////////////////////////////////////////
-/**
- * Convenient function to get size of an object
- *
- * @param {object} obj
- * @returns {number} *
- */
-//////////////////////////////////////////////////////////////////////////////
-Object.size = function (obj) { // jshint ignore: line
+geo.inherit.func = function () {
```

Comment:
Curious. Why you think its bad. JS supposed be extended. A similar code is suggested here: http://stackoverflow.com/questions/5223/length-of-a-javascript-object-that-is-associative-array